src/App.css

```diff
@@ -35,3 +34,57 @@
   from { transform: rotate(0deg); }
   to { transform: rotate(360deg); }
 }
+
+/****** HOME PAGE STYLES ******/
+
+.raisedButton {
+    width: 200px;
+}
+.accountIcon {
+    margin-right: 2em;
+}
+
+.logo {
+    margin-left: 4em;
+    width: 4em;
+    padding: 5px;
+}
+
+.appBar {
+
+}
+
+.sidebar {
+    width: 12%;
+    height: 100vh;
+    border-right: 1px solid #dbdbdb;
+    float: left;
+}
+
+.mainContent {
+    display: inline-block;
+    margin: auto;
+    max-width: 88%;
+    height: 100%;
+    /*background-color: #dbdbdb;*/
+}
+
+.searchCard {
+    /*position: absolute;*/
+	display: inline-block !important;
+	margin: auto !important;
+	margin-left: 4em !important;
+    margin-bottom: 1em !important;
+}
+
+.searchIcon {
+	color: #38b5e6 !important;
```

Comment:
Is the css framework that bad that we have to `!important` a color? I really hope not. 

http://www.material-ui.com/#/customization/styles

Not sure if you grabbed the info from that page, but it looks like `!important` should only really be needed if you are overriding a inline style that already exists.

Reply:
For icon colors, you do in fact need to use !important, sadly. There are some Material UI component attributes that let you do certain things easily, like size and color, but I prefer to keep as much styling as possible in a css file. 

What is your opinion on using !important to keep our styles in the css files vs doing everything inline?

Reply:
My preference is inline. In general I try to reserve using css for only media queries.